### DISCUSSION TITLE
Dry-Run Issue

### DISCUSSION THREAD

**Discussion by Frizlab** [fluxcd/flux2#1933] [Q&A] [answered]
After upgrading flux2 (from `v0.17.2` to `v0.18.2`) I now get this message for one of my Kustomization:
```text
Elasticsearch/infra/es-logs-eck dry-run failed, reason: BadRequest, error: admission webhook "elastic-es-validation-v1.k8s.elastic.co" does not support dry run
```

I’m not sure what that means, but I guess now Flux2 tests the Kustomization before actually applying it. Is there a way to disable this test for objects not supporting dry-run? I read the upgrade notes, but did not find anything about this.

Thanks for the help.

**Comment by stefanprodan** [comment, marked as answer]
Please see https://github.com/fluxcd/flux2/issues/1927
